### PR TITLE
Add all Google Web Components

### PR DESCRIPTION
**Pull request by VandeurenGlenn**

### CODE REVIEW

File: snippets/google-youtube.cson [hunk 1]
```diff
@@ -0,0 +1,14 @@
+".text.html":
+  "google-youtube":
+    "prefix": "gy"
```
Comment (robdodson) [inline snippets/google-youtube.cson]: gyt

File: snippets/google-youtube-upload.cson [hunk 2]
```diff
@@ -0,0 +1,17 @@
+".text.html":
+  "google-youtube-upload":
+    "prefix": "gyu"
```
Comment (robdodson) [inline snippets/google-youtube-upload.cson]: gytu

File: snippets/google-youtube-api.cson [hunk 3]
```diff
@@ -0,0 +1,9 @@
+".text.html":
+  "google-youtube-api":
+    "prefix": "gya"
```
Comment (robdodson) [inline snippets/google-youtube-api.cson]: gyta

File: snippets/google-chart.cson [hunk 4]
```diff
@@ -0,0 +1,12 @@
+".text.html":
+  "google-chart":
+    "prefix": "gc"
```
Comment (robdodson) [inline snippets/google-chart.cson]: gch

File: snippets/google-feeds.cson [hunk 5]
```diff
@@ -0,0 +1,10 @@
+".text.html":
+  "google-feeds":
+    "prefix": "gf"
```
Comment (robdodson) [inline snippets/google-feeds.cson]: gfe

File: snippets/google-map.cson [hunk 6]
```diff
@@ -0,0 +1,20 @@
+".text.html":
+  "google-map":
+    "prefix": "gm"
```
Comment (robdodson) [inline snippets/google-map.cson]: gma

File: snippets/google-maps-api.cson [hunk 7]
```diff
@@ -0,0 +1,11 @@
+".text.html":
+  "google-maps-api":
```
Comment (robdodson) [inline snippets/google-maps-api.cson:2]: gmaa

File: snippets/google-sheets.cson [hunk 8]
```diff
@@ -0,0 +1,13 @@
+".text.html":
+  "google-sheets":
+    "prefix": "gs"
```
Comment (robdodson) [inline snippets/google-sheets.cson]: gsh

**Comment by robdodson** [comment]
Wow cool! I added a few comments, basically making sure each snippet is at least 3 characters long. I figure since we're injecting so many it might be good to require a minimum length in case someone already has `gc` as a snippet?

**Comment by VandeurenGlenn** [comment]
Thanks, good point! I'll make the changes when I'm home.

File: snippets/google-maps-api.cson (hunk 7 above)
Comment (VandeurenGlenn) [inline snippets/google-maps-api.cson:2]: gmaapi

File: snippets/google-youtube-api.cson (hunk 3 above)
Comment (VandeurenGlenn) [inline snippets/google-youtube-api.cson]: gytapi

**Comment by VandeurenGlenn** [comment]
What do you think about my comments for adding api at the end?

File: snippets/google-plusone-api.cson [hunk 9]
```diff
@@ -0,0 +1,9 @@
+".text.html":
+  "google-plusone-api":
+    "prefix": "gpa"
```
Comment (VandeurenGlenn) [inline snippets/google-plusone-api.cson]: gpapi

File: snippets/google-realtime-api.cson [hunk 10]
```diff
@@ -0,0 +1,9 @@
+".text.html":
+  "google-realtime-api":
+    "prefix": "gra"
```
Comment (VandeurenGlenn) [inline snippets/google-realtime-api.cson]: grapi

**Comment by robdodson** [comment]
@VandeurenGlenn I think adding api to the end is fine.

**Comment by VandeurenGlenn** [comment]
@robdodson Ok, I'll make the changes, happy new year!

**Comment by robdodson** [comment]
Awesome, thanks!

**Comment by VandeurenGlenn** [comment]
You're welcome!

**Pull request merged**